File: 2000 98 & ME/QuickStart/howto/samples/Services/MessageQueue/sendobject/vb/customer.vb
Imports System

Public Class Customer
	protected last As String
	protected first As String
	
	public Property LastName as string
                get
			return me.last
                End get
                set
			Me.last = value
                End set
        end property 
        
	public property FirstName as string 
		get
			return me.first
		end get 
		set
			me.first = value
		end set
        end property

	public readonly property FullName as string
		get
			return (FirstName + LastName)
		end get
	end property
End Class

File: 2000 98 & ME/QuickStart/howto/samples/Services/MessageQueue/sendobject/customer.cs
using System;

public class Customer
{
	protected string lastName;
	protected string firstName;
	
	public string LastName{
		get{
			return this.lastName;
		}
		set{
			this.lastName = value;
		}
	}

	public string FirstName{
		get{
			return this.firstName;
		}
		set{
			this.firstName = value;
		}
	}	

	public string FullName{
		get{
			return (FirstName + LastName);
		}
	}
}